### PR TITLE
Update ToC to better match docplan spreadsheet

### PR DESCRIPTION
**Pull request by indie**
- Order ToC as requested by PMs
- Add content from `collab_ngai` to master
- Clean-up other minor wording and linking issues.

### CODE REVIEW

**Comment by indie** [comment]
Removed review request from @avijit-nervana 's team since we just shortened the title of that page, as requested by @bdalby for consistency in the menu.

**Comment by indie** [comment]
Adding the preview of doc build here as well   ngraph_preview_toc

File: doc/sphinx/source/core/quantization.rst [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+accuracy. INT8 is one kind of quantization. Compared with 32-bit floating point 
+(FP32), using arithmetic with lower precision, such as INT8, to calculate 
+weights and activation requires less memory.
+
+
+Implementing a quantized model with nGraph
+------------------------------------------
+
+To implement a quantized model with nGraph, provide a partially (or fully)
+quantized FP32 model (where the convolution layer in the model is replaced
```
Comment (diyessi) [inline doc/sphinx/source/core/quantization.rst]: Remove `FP32`

File: doc/sphinx/source/frameworks/other/index.rst [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+If your nGraph build is a Neural Network configured on Clear Linux\* OS 
+for Intel® Architecture, and it includes at least one older CPU, the 
+`following article may be helpful`_.
+
+
+Training Deep Neural Networks
+-----------------------------
+
+Before tweaking various environment variables, be aware that how the computation 
+gets executed depends upon the ordering of the data format that the model is
```
Comment (diyessi) [inline doc/sphinx/source/frameworks/other/index.rst]: `depends on the data layout that the model is`

File: doc/sphinx/source/frameworks/other/index.rst [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+for Intel® Architecture, and it includes at least one older CPU, the 
+`following article may be helpful`_.
+
+
+Training Deep Neural Networks
+-----------------------------
+
+Before tweaking various environment variables, be aware that how the computation 
+gets executed depends upon the ordering of the data format that the model is 
+using. ``NHWC`` and ``NCHW`` are the two more common layouts in Deep Learning
```
Comment (diyessi) [inline doc/sphinx/source/frameworks/other/index.rst]: `... are common layouts...`

**Comment by indie** [comment]
0 27_preview_toc

**Pull request merged**